dalesbred/src/main/java/org/dalesbred/query/SqlQuery.java

```diff
@@ -44,11 +46,17 @@
 
     private final @NotNull List<?> args;
 
+    private @Nullable Integer fetchSize;
+
+    private @Nullable Integer fetchDirection;
```

Comment:
Internally JDBC uses integer constants (because the API predates enums), but I'd add a new enumeration type for this:

```java
public enum FetchDirection {
    FORWARD(ResultSet.FETCH_FORWARD), 
    REVERSE(ResultSet.FETCH_REVERSE), 
    UNKNOWN(ResultSet.FETCH_UNKNOWN);

    private final int jdbcCode;

    FetchDirection(int jdbcCode) {
        this.jdbcCode = jdbcCode;
    }

    public int getJdcbCode() { 
        return jdbcCode;
    }
}
```